doc/source/rllib/rllib-training.rst

```diff
@@ -1182,18 +1182,19 @@ on the ``evaluation_duration_unit`` setting, which can be either "episodes" (def
 
 
 Note: When using ``evaluation_duration_unit=timesteps`` and your ``evaluation_duration`` setting is NOT dividable
-by the number of evaluation workers (configurable via ``evaluation_num_workers``), RLlib will round up the number of timesteps specified to the nearest whole number of timesteps that is divisible by the number of evaluation workers.
+by the number of evaluation workers (configurable via ``evaluation_num_workers``), RLlib will round up the number of
+timesteps specified to the nearest whole number of timesteps that is divisible by the number of evaluation workers.
 
 Before each evaluation step, weights from the main model are synchronized to all evaluation workers.
 
-By default, the evaluation step is run right after the respective training step. For example, for
-``evaluation_interval=2``, the sequence of events is: ``train, train, eval, train, train, eval, ...``.
-For ``evaluation_interval=1``, the sequence is: ``train, eval, train, eval, ...``.
+By default, the evaluation step (if there is one in the current iteration) is run right **after** the respective training step.
+For example, for ``evaluation_interval=2``, the sequence of events is: ``train-step, train-step, eval-step, train-step, train-step, eval-step, ...``.
+For ``evaluation_interval=1``, the sequence is: ``train-step, eval-step, train-step, eval-step, ...``.
 
 However, it is possible to run evaluation in parallel to training via the ``evaluation_parallel_to_training=True``
-config setting. In this case, both training- and evaluation steps are run at the same time via threading.
+config setting. In this case, both training- and evaluation steps are run at the same time via multi threading.
 This can speed up the evaluation process significantly, but leads to a 1-iteration delay between reported
```

Comment:
Can you elaborate this with an example? e.g. training metrics in iteration t are shown aligned with evaluation metrics at time t-1?

Reply:
done